modules/creating-instance-oci.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-instance-oci_{context}"]
+= Completing the remaining Assisted Installer steps
+
+You can use the {oci-first} Compute Service to create a virtual machine (VM) instance on {oci}. This instance can then automatically attach to a virtual network interface card (VNIC) in the virtual cloud network (VNC) subnet. On specifying the IP address of your {product-title} cluster in the custom manifest template files, the {oci} instance can comunicate with your cluster over the VNC. 
+
+.Prerequisites
+
+* You completed all the mandatory Assisted Installer steps. See the "Completing the remaining Assisted Installer steps" section.
+
+.Procedure
+
+. Open the custom manifest template, `oci-ccm.yml`, and replace the cloud configuration section of the template file with the configuration you previously copied from the **Logs** section on {oci}. Perform the same steps to update the configuration of the CSI driver in the `oci-csi.yml` file.
++
+The following example replaced the `region:` to `rateLimitBucketWrite` parameters of the `oci-csi.yml` file with the configuration from the `oci_ccm_config` cloud configuration on {oci}. 
++
+[source,yaml]
+----
+# …
+config.yaml: |
+   auth: 
+     region: <region_name>
+     useInstancePrincipals: true
+     compartment: <compartment_ocid>
+   vcn: <virtual_cloud_network_ocid>
+   loadBalancer:
+      subnet1: <subnet_ocid>
+      securityListManagementMode: Frontend
+   rateLimiter:
+     rateLimitQPSRead: 20.0
+     rateLimitBucketRead: 5
+     rateLimitQPSWrite: 20.0
+     rateLimitBucketWrite: 5
+# …
+----
+
+. Go to **Services > Instances** on {oci}. Select your compartment, and then click the **Create instance** button. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Compute/Tasks/launchinginstance.htm#top[Creating an instance] in the Oracle documentation. 
+
+. Add users to the instance. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Compute/Tasks/addingusers.htm#Adding_Users_on_an_Instance[Adding users to an instance] in the Oracle documentation. 
```

Comment:
@faermanj . This adding users steps seems like a post-installation step. WDYT? If so, I would need to create a new section just before the troubleshooting section. However, this new section is very small so maybe I could add others task or perhaps add a "Next steps" section and link to Oracle docs?